### PR TITLE
Update image-no-text-0va7u6.md

### PR DESCRIPTION
**Pull request by ChrisLoiselle**
Per discussion on 6/16/2022 call with AGWG ACT Task Force to update HTML graphics contain no text rule link wording.

Need for Call for Review:
None.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by Jym77 — DISMISSED** [review]

File: _rules/image-no-text-0va7u6.md [hunk 1]
```diff
@@ -34,12 +34,16 @@ acknowledgments:
...
+- **language**: the [visible pixels][visible] of the test target do not contain text expressing anything in a [human language]
+
+## Exceptions 
+
+For each test target, at least one of the following exceptions is true:
 
 - **essential**: the [visible pixels][visible] of the test target contain text for which its presentation is [essential][]; or
-- **language**: the [visible pixels][visible] of the test target do not contain text expressing anything in a [human language][]; or
 - **significant**: the [visible pixels][visible] of the test target contain text and the text is not a significant part of the [visible pixels][visible] of the test target; or
 - **decoration**: the [embedded image][] the test target belongs to is [purely decorative][].
```
[the remark this answers is not in the document]
Comment (ChrisLoiselle) [reply _rules/image-no-text-0va7u6.md:48]: Hi @Jym77 ,

Could you confirm you are ok with:

For each test target, its [visible pixels][visible] do not contain text expressing anything in a [human language], except if at least one of the following is true:
- **essential**: the [visible pixels][visible] of the test target contain text for which its presentation is [essential][]; or
- **significant**: the [visible pixels][visible] of the test target contain text and the text is not a significant part of the visible part of the image or
- **decoration**: the [embedded image][] the test target belongs to is [purely decorative][].

If so, I'll update the pull request . Thanks ! Chris

[the remark this answers is not in the document]
Comment (Jym77) [reply _rules/image-no-text-0va7u6.md:48]: Yes, that looks good 👍

**Review by HelenBurge — APPROVED** [review]

File: _rules/image-no-text-0va7u6.md [hunk 2]
```diff
@@ -34,13 +34,17 @@ acknowledgments:
 
 This rule applies to any [rendered image resources][] in a [web page][].
 
-## Expectation
+## Expectations
```
Comment (Jym77) [inline _rules/image-no-text-0va7u6.md]:
```suggestion
## Expectation
```

File: _rules/image-no-text-0va7u6.md [hunk 3]
```diff
@@ -34,13 +34,17 @@ acknowledgments:
...
 
-For each test target, at least one of the following is true:
+For the test target, the expectation is that :
+- **language**: the [visible pixels][visible] of the test target do not contain text expressing anything in a [human language]
+
+## Exceptions
```
Comment (Jym77) [inline _rules/image-no-text-0va7u6.md]:
```suggestion
```

**Review by Jym77 — DISMISSED** [review]

**Review by wcreedle — APPROVED** [review]

**Review by wcreedle — APPROVED** [review]

File: _rules/image-no-text-0va7u6.md [hunk 4]
```diff
@@ -36,11 +36,11 @@ This rule applies to any [rendered image resources][] in a [web page][].
...
 ## Expectation
 
-For each test target, at least one of the following is true:
+
+For each test target, its [visible pixels][visible] do not contain text expressing anything in a [human language], except if at least one of the following is true:
 
 - **essential**: the [visible pixels][visible] of the test target contain text for which its presentation is [essential][]; or
-- **language**: the [visible pixels][visible] of the test target do not contain text expressing anything in a [human language][]; or
-- **significant**: the [visible pixels][visible] of the test target contain text and the text is not a significant part of the [visible pixels][visible] of the test target; or
+- **significant**: the [visible pixels][visible] of the test target contain text and the text is not a significant part of the visible part of the image or
```
Comment (kengdoj) [inline _rules/image-no-text-0va7u6.md]:
```suggestion
- **not significant**: the [visible pixels][visible] of the test target contain text and the text is not a significant part of the visible part of the image or
```

File: _rules/image-no-text-0va7u6.md [hunk 5]
```diff
@@ -36,11 +36,11 @@ This rule applies to any [rendered image resources][] in a [web page][].
...
 - **decoration**: the [embedded image][] the test target belongs to is [purely decorative][].
```
Comment (kengdoj) [inline _rules/image-no-text-0va7u6.md]:
```suggestion
- **decoration**: the test target belongs to an [embedded image][] that is [purely decorative][].
```

**Review by wcreedle — DISMISSED** [review]

**Review by kengdoj — APPROVED** [review]

**Review by wcreedle — APPROVED** [review]

File: _rules/image-no-text-0va7u6.md [hunk 6]
```diff
@@ -36,12 +36,12 @@ This rule applies to any [rendered image resources][] in a [web page][].
 
 ## Expectation
 
-For each test target, at least one of the following is true:
+
+For each test target, its [visible pixels][visible] do not contain text expressing anything in a [human language], except if at least one of the following is true:
 
 - **essential**: the [visible pixels][visible] of the test target contain text for which its presentation is [essential][]; or
```
Comment (WilcoFiers) [inline _rules/image-no-text-0va7u6.md:42]: I'm not so sure we should use "essential" here. It's sort of out of scope of this PR, but it's very relevant in moving this rule forward. We had a discussion about "essential" last week in ACT TF and our impression there was that it was too ambiguous to be used in an expectation.

Comment (ChrisLoiselle) [reply _rules/image-no-text-0va7u6.md:42]: Hi @WilcoFiers , your points seem to point at both including and excluding it. I'm happy to have a discussion to decide on what we are doing within the most recent submission for updates.

Comment (WilcoFiers) [reply _rules/image-no-text-0va7u6.md:42]: I'll open an issue. This is not related to your PR. Feel free to move forward with that. https://github.com/act-rules/act-rules.github.io/issues/1915

**Review by WilcoFiers — APPROVED** [review]

File: _rules/image-no-text-0va7u6.md (hunk 6 above)
Comment (ChrisLoiselle) [reply _rules/image-no-text-0va7u6.md:42]: So I can resolve this conversation ? I see https://github.com/act-rules/act-rules.github.io/issues/1915 however not sure what to do with https://github.com/act-rules/act-rules.github.io/pull/1873 currently.

**Pull request merged**